dmutils/email.py

```diff
@@ -40,6 +43,7 @@ def send_email(to_email_addresses, email_body, api_key, subject, from_email, fro
             'track_clicks': False,
             'auto_text': True,
             'tags': tags,
+            'metadata': metadata,
```

Comment:
I think it could be anything that's serialisable to JSON. Dicts probably make the most sense.